lib/test_highlevel.ml

```diff
@@ -12,78 +12,100 @@
  * GNU Lesser General Public License for more details.
  *)
 
-open OUnit
+open Xapi_stdext_std
```

Comment:
Try to avoid opening modules it can be harmful as it can shadow functions, you can use `module Std = Xapi_stdext_std` if you need a short name.

Please read https://medium.com/@huund/do-not-open-6e5d3070c118 for a more in-depth explanation

Reply:
I think this would be a good time to point out https://github.com/lindig/ocaml-style for some background reading

Reply:
If this was just for the `Xstringext.String.strip` below, then I don't think we actually need it at all (nor the lib dependency).

Reply:
@edwintorok it would be nice if there was some tool to format the code according to this style automatically

Reply:
There is https://github.com/ocaml-ppx/ocamlformat, it is quite opinionated though. Recommend using it for completely newly written modules, but not for changing existing code, it would make backporting a lot harder.